specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "creationParams",
+                        "in": "body",
+                        "description": "Client supplied parameters.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/AttestationServiceCreationParams"
+                        },
+                        "x-ms-parameter-location": "method"  
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Created or updated attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "201": {
+                        "description": "Created or updated attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "409": {
+                        "description": "Duplicate attestation name.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Error result from Attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResult"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "AttestationProvider"
+                ],
+                "operationId": "AttestationProviders_Delete",
+                "description": "Delete Attestation Service.",
+                "x-ms-examples": {
+                    "AttestationProviders_Delete": {
+                        "$ref": "./examples/Delete_AttestationProvider.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "name": "providerName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9-]{3,24}$",
+                        "description": "Name of the attestation service"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
```

Comment:
Remove 404 and 500 errors, but you need to add a 204.  From the RPC:
_The resource provider can return 200 (OK) or 204 (NoContent) to indicate that the operation completed successfully. A 200 (OK) should be returned if the object exists and was deleted successfully; and a 204 (NoContent) should be used if the resource does not exist and the request is well formed._

Reply:
remove 404 and 500 errors and add
                    "200": {
                        "description": "Resource exists and was deleted successfully"
                    },
                    "204": {
                        "description": "Resource does not exist",
                    },